code/src/index.css

```diff
@@ -11,3 +20,144 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* HEADER */
+
+.header {
+  display: flex;
+  justify-content: center;
+  padding-top: 60px;
+}
+
+.logo {
+  height: 100px;
+}
+
+/* MAIN */
+
+.main {
+  display: flex;
+  margin: 50px 100px;
+  align-items: center;
+  flex-direction: column;
+  font-family: "Menlo Regular";
+}
+
+/* FORM & MESSAGE CONTAINER */
+
+.form,
+.message-container {
+  display: flex;
+  width: 500px;
+  flex-direction: column;
+  border: 1px solid black;
+  padding: 15px;
+  box-shadow: 7px 7px;
+}
+
+.form {
+  background-color: #f2f0f0;
+}
+
+.message-container {
+  background-color: white;
+  margin-top: 30px;
+  justify-content: space-between;
+}
+
+/* FORM ELEMENTS */
+
+.new-message,
+.form-heading {
+  width: 450px;
+  margin: 10px;
+  font-family: inherit;
+}
+
+.new-message {
+  height: 60px;
+  resize: none;
+  padding: 10px;
+}
+
+.counter {
+  margin: 12px;
+}
+
+.submit-button {
+  border-radius: 20px;
+  border: 0px;
+  width: 300px;
+  height: 40px;
+  background-color: #ffadad;
+  margin: 10px;
+  font-family: inherit;
+  cursor: pointer;
+}
+
+.heart-icon {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.error-msg {
+  margin: 0;
+}
+
+/* MESSAGE LIST ELEMENTS */
+
+.message {
+  overflow-wrap: break-word;
+}
+
+.likes-container {
+  display: flex;
+  align-items: baseline;
+}
+
+.likes-container p {
+  margin: 0;
+  color: #cdcdcd;
+  margin-left: 5px;
+}
+
+.time-hearts-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.heart {
+  background-color: #eaeaea;
+  border: 0px;
+  border-radius: 50%;
+  width: 43px;
+  height: 43px;
+  text-align: center;
+  margin: 0;
+  cursor: pointer;
+  transition: font-size 2s ease-in-out 1s;
```

Comment:
I don't  know way, but I can't see this effect on the heart, it looks like it happens faster and with out the easings. And it looks a bit like the heart "pushes" the background/button up a bit, I'm not sure if this is a good solution but maybe both the heart and the background could grow larger together?